lightrag/llm.py

```
@@ -455,9 +470,16 @@ async def lmdeploy_model_if_cache(
     return response
 
 
+class GPTKeywordExtractionFormat(BaseModel):
+    high_level_keywords: List[str]
+    low_level_keywords: List[str]
+
+
 async def gpt_4o_complete(
-    prompt, system_prompt=None, history_messages=[], **kwargs
+    prompt, system_prompt=None, history_messages=[], keyword_extraction=False, **kwargs
```

Ruff (0.8.0)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)